shortcodes/epfl_news/view.php

```diff
@@ -15,8 +15,11 @@
   }
 ?>
 
-<div class="container-full">
+<?php if ("1" == $template): ?>
   <div class="container">
+<?php else: ?>
+  <div class="container-full">
+<?php endif ?>
```

Comment:
Tsss, on avait pas dit qu'on mettait ?!?
> if(...)
{
//the über code
}

Au lieu des `endif` ?

Reply:
Hier c'était pour un switch dans du code exclusivement php. Ici on est dans un template. Pour moi, on gagne en lecture que la balise de fin soit un endif plutôt qu'un } surtout lorsque il y a imbrication de différents if foreach et autres.

Reply:
Ok pour ta réponse.